packages/app-headless-cms/src/admin/plugins/fieldRenderers/dynamicZone/TemplateGallery.tsx

```diff
@@ -67,29 +67,23 @@ const Gallery = makeDecoratable("TemplateGalley", (props: TemplateGalleryProps)
     );
 });
 
-function withDecoratorFactory<T extends DecoratableComponent>(Component: T) {
-    return Object.assign(Component, {
-        createDecorator: createDecoratorFactory<{ modelIds?: string[] }>()(
-            Component,
-            decoratorProps => {
-                const { model } = useModel();
+export type ShouldRender = { modelIds?: string[] };
```

Comment:
Check this file for the `TemplateGalley` string 🙃

Reply:
😂 Nice find, but functionally it doesn't affect anything, it's just a displayName for components. Will fix.